packages/auth/src/interaction/routes.ts

```diff
@@ -123,7 +123,7 @@ async function startInteraction(
   const { config, grantService } = deps
   const grant = await grantService.getByInteractionSession(interactId, nonce)
 
-  if (!grant) {
+  if (!grant || grant.state !== GrantState.Processing) {
```

Comment:
```suggestion
  if (!grant || grant.state !== GrantState.Pending) {
```
The eCommerce postman example wasn't working for me when starting the interaction, this is what I needed (and I think makes sense from the GNAP diagram).

The difference between Processing and Pending isn't that clear until I saw the diagram 😅 . Now that we don't use `Processing` anywhere, I was hoping we could remove it, but I guess otherwise we wouldn't be able to have a "starting" state...

Reply:
A direction I was thinking of taking the AS in the future is to have an explicit `process` function on the Grant service and have it behave a bit more like a state machine. So something like `create` or `finishInteraction` would leave the grant in `Processing` and then call `process` to determine what state to transition the grant next.

I don't think there will be enough information with our current schema to do this until I finish https://github.com/interledger/rafiki/issues/1655

Reply:
I'll fix this, though.